src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -56,7 +56,6 @@ CouplingSchemeConfiguration::CouplingSchemeConfiguration(
       ATTR_LIMIT("limit"),
       ATTR_MIN_ITERATIONS("min-iterations"),
       ATTR_NAME("name"),
-      ATTR_TIMESTEP_INTERVAL("timestep-interval"),
```

Comment:
Can we really remove this line? Or do we have to find an appropriate replacement? I.e. `ATTR_TIMEWINDOW_INTERVAL`